packages/client-core/src/systems/ui/UserMenuView/index.tsx

```diff
@@ -76,7 +77,7 @@ const AvatarContextMenu = () => {
 
   useEffect(() => {
     if (detailState.id.value !== '') {
-      const tappedUser = Engine.instance.worldNetworkState.peers.values().find((peer) => peer.userId === detailState.id.value)
+      const tappedUser = Array.from(Engine.instance.worldNetworkState.peers.get({ noproxy: true }).values).find((peer) => peer.userId === detailState.id.value)
```

Comment:
can avoid the noproxy get by using worldNetwork instead of worldNetworkState, it returns the same thing